galleries/examples/widgets/menu.py

```diff
@@ -5,19 +5,21 @@
 
 Using texts to construct a simple menu.
 """
+
+from dataclasses import dataclass
+
 import matplotlib.pyplot as plt
 
 import matplotlib.artist as artist
 import matplotlib.patches as patches
 
 
+@dataclass
 class ItemProperties:
-    def __init__(self, fontsize=14, labelcolor='black', bgcolor='yellow',
-                 alpha=1.0):
-        self.fontsize = fontsize
-        self.labelcolor = labelcolor
-        self.bgcolor = bgcolor
-        self.alpha = alpha
+    fontsize: float = 14
+    labelcolor: str = 'black'
+    bgcolor: str = 'yellow'
```

Comment:
Couldn't these be any matplotlib color spec type?

Reply:
True, though I'm not sure this example ever would.